finality-aleph/src/lib.rs

```diff
@@ -38,6 +38,7 @@ mod nodes;
 mod party;
 mod session;
 mod session_map;
+pub mod status;
```

Comment:
I'm not convinced this is necessary as a separate module, since it just very thinly pack a `ticker`. Perhaps just move the `STATUS_TICKER_DELAY` (maybe rename to `STATUS_REPORT_INTERVAL`?) to this `lib.rs` file and crate plain tickers everywhere?

If not, then at least remove the `pub` here, it's unnecessary.

Reply:
removed it